Feature: localized release notes (#401)
codemagic-ci-cd/codemagic-docs: content/publishing/publish-release-notes.md

    * For Google Play it will be published under `en-US` language localization code.

{{<notebox>}}

> For App Store Connect supported languages and codes are listed [here](https://developer.apple.com/documentation/appstoreconnectapi/betabuildlocalizationcreaterequest/data/attributes). For Google Play Console supported languages and codes are listed [here](https://support.google.com/googleplay/android-developer/table/4419860?hl=en).

Reviewer: ```suggestion
For App Store Connect, supported languages and codes are listed [here](https://developer.apple.com/documentation/appstoreconnectapi/betabuildlocalizationcreaterequest/data/attributes). For Google Play Console, supported languages and codes are listed [here](https://support.google.com/googleplay/android-developer/table/4419860?hl=en).
```

{{</notebox>}}

2. Create a `release_notes_<language_localization_code>.txt` file for every language used, e.g. `release_notes_en-GB.txt`, `release_notes_it.txt`, and add them to the root of your project.
    * Release notes with `en-US` language code will be published to email and Slack in case file with `en-US` language code exists. If not, the first found release notes will be published.

>     * For App Store Connect and Google Play, all the release notes will be published with corresponding language codes, ommiting language codes that are not supported.

Reviewer: ```suggestion
    * For both App Store Connect and Google Play, only the release notes with the supported language codes will be published, omitting language codes that are not supported.
```

3. Create a `release_notes.json` file with the following content:

    ```json
    [
        {
            "language": "en-GB",
            "text": "British English release notes text"
        },
        {
            "language": "en-US",
            "text": "The US Engish release notes text"
        }
    ]
    ```

    Add this file to the root of your project. Notes with missing `language` or `text` fields will not be taken into account.

    * Release notes with `en-US` language code will be published to email and Slack, given that a file with `en-US` language code exists. If not, the first release notes will be published.

>     * For App Store Connect and Google Play, all the found release notes will be published with corresponding language codes, omiting language codes that are not supported.

Reviewer: ```suggestion
    * For both App Store Connect and Google Play, only the release notes with the supported language codes will be published, omitting language codes that are not supported.
```
